spec/poltergeist_helper.rb

```diff
@@ -0,0 +1,7 @@
+require 'rails_helper'
```

Comment:
Is there any reason not to move this into `rails_helper.rb`? It would be more in line with other projects I've worked on.